tests/helpers/rabbitMQ.ts

```diff
@@ -0,0 +1,27 @@
+/* tslint:disable:no-console */
+/* tslint:disable:no-relative-imports */
+import { connect } from 'amqplib'
+
+export const RabbitMQ = async (rabbitUrl: string) => {
+  console.log(`Connecting to RabbitMQ...`)
+  const connection = await connect(rabbitUrl)
+
+  return {
+    consume: (exchange: string, callback: (message: any) => Promise<void>) => consume(connection, exchange, callback),
```

Comment:
There's an interface for Message in the ampqlib type declarations. Can you try `ConsumeMessage`?